README.md

```diff
@@ -1,7 +1,7 @@
-1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_center_spider_DOM/)
+1. Replace https://github.com/sinner1993/js_center_spider_DOM with your Github username in the link
+    - [DEMO LINK](https://sinner1993.github.io/js_center_spider_DOM/)
```

Comment:
This demo link must be in the PR description, not the repository link 
<img width="639" alt="image" src="https://github.com/user-attachments/assets/653afc16-88f8-4d31-b3ea-56d2f9502fca">